test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,9 @@
+require 'test_helper'
+
+class HomeControllerTest < ActionDispatch::IntegrationTest
+  test "should get index" do
+    get get root_url
```

Comment:
does this pass with a double `get`?

Reply:
Hmm... I get the same error (below) whether there's a double or single get. 

Since this test was autogenerated and we're going to discuss integration tests instead of controller tests, should I delete this for now (as I deleted the profile_controller_test.rb file)?

Error:
HomeControllerTest#test_should_get_index:
ActiveRecord::RecordNotUnique: PG::UniqueViolation: ERROR:  duplicate key value violates unique constraint "index_users_on_email"
DETAIL:  Key (email)=(MyString) already exists.
: INSERT INTO "users" ("name", "about", "email", "created_at", "updated_at", "id") VALUES ('MyString', 'MyText', 'MyString', '2018-01-31 15:49:23.730070', '2018-01-31 15:49:23.730070', 298486374)

Reply:
👍 to deleting this. I'm assuming that there's a fixture somewhere (`test/fixtures` maybe?) that's creating multiple users with the same email address. We'll get FactoryBot setup on this and walkthrough that when you're ready.